lib/float.gd

```diff
@@ -180,12 +171,56 @@ DeclareAttribute("Gamma", IsFloat);
 ##       if <A>x&gt;0</A>.
 ##   </Description>
 ## </ManSection>
-## <#/GAPDoc>
 ##
+## <ManSection>
+##   <Attr Name="SinCos" Arg="x"/>
+##   <Returns>The list <C>[sin(x), cos(x)]</C>.</Returns>
+##   <Description>
+##       The function returns a list with <C>sin</C> and <C>cos</C> of <A>x</A>.
+##   </Description>
+## </ManSection>
+##
+## <ManSection>
+##   <Oper Name="Atan2" Arg="y x"/>
+##   <Returns>The polar angle of <A>(x, y)</A> in the plane as float.</Returns>
+##   <Description>
+##        Returns the principal value of the argument (polar angle) of (x, y) in the plane.
+##        The returned value will always be in <M>(-\pi , \pi]</M> and is not defined on <A>(0,0)</A>.
+##        This function is defined in accordance with IEEE 1788-2015 and imported from IEEE 754.
+##   </Description>
+## </ManSection>
+##
+## <ManSection>
+##   <Attr Name="Log1p" Arg="x"/>
+##   <Attr Name="Expm1" Arg="x"/>
+##   <Returns>The natural logarithm of <A>x+1</A> or exponential <M>-1</M> of <A>x</A> respectively.</Returns>
+##   <Description>
+##       The first function <C>Log1p</C> returns the natural logarithm <M>log(x+1)</M>.
+##
+##       <P/> The second function <C>Expm1</C> returns the exponential function <M>exp(x)-1</M>
+##
+##       <P/> These two functions are inverse to each other.
+##   </Description>
+## </ManSection>
+##
+## <ManSection>
+##   <Oper Name="Erf" Arg="x"/>
+##   <Returns>The error function given by the Gaussian integral</Returns>
+##   <Description>
+##        Returns the error function imported from IEEE 754 given by the formula:<Br/>
+##        <M> Erf(x) := \frac{2}{\sqrt{\pi}} \int_{0}^{x} exp(- t^2 ) dt </M>
```

Comment:
Maybe use display math instead of a line break?
```suggestion
##        Returns the error function imported from IEEE 754 given by the formula:
##        <Display> Erf(x) := \frac{2}{\sqrt{\pi}} \int_{0}^{x} exp(- t^2 ) dt </Display>
```